v19.2/cockroach-demo.md

```diff
@@ -229,6 +230,15 @@ $ cockroach demo --with-load
 
 This command starts a demo cluster with the `movr` database preloaded and then inserts rows into each table in the `movr` database. You can monitor the workload progress on the [Admin UI](admin-ui-overview-dashboard.html#sql-queries).
 
+### Run `cockroach demo` with the [Geo-Partitioned Replicas](topology-geo-partitioned-replicas.html) topology pattern
```

Comment:
Having the link in the heading's text causes the page's TOC entry to have BIG TEXT (but still lowercase) after the link.  Suggest making a shorter heading like "Run cockroach demo with geo-partitioned replicas" and letting the text below do the work of linking since the info is already there.  Our style guide is silent on this issue, perhaps I'll make a PR.